### PR TITLE
Make canned meat recipes use the right amount of meat

### LINKED ISSUE #32743: Canning meat results in extradimensional foodstuffs
# Describe the bug

When canning meat in a 0.5L jar, the result is 'canned meats (2/1)'

# Steps To Reproduce

Ingredients:

- Glass jar (0.5L)
- Canning pot
- 2 x chunk of meat

Place chunks of meat in the glass jar, add water to cover the meat, and cook in the canning pot until grey. *Do not add salt* as this may result in the canned meat being appetising. At the end of the canning process, screw down the lid. Observe your `canned meats (2/1)` is able to defy space and time and contain more meat than the jar is able to hold!

# Expected behavior

Either 'canned meats (2/2)', or being able to only can a single chunk in a 0.5L jar, thus giving 'canned meats (1/1)'.

# Versions and configuration

 - OS: Ubuntu 16.04 LTS
 - Game Version: 03c92080c77055ddd22223c1b122ce41291de856
 - Graphics version: Tiles

### PR DESCRIPTION
**Pull request by brannamm**
#### Summary
```SUMMARY: Bugfixes "Make canned meat recipes use the right amount of meat"``` 
#### Purpose of change
Fixes #32743
#### Describe the solution
The amount of meat and the volume of the container for the canned meat recipes in glass jar and 3L glass jar did not add up to the same value before this change and now they do.

#### Describe alternatives you've considered
Halving the volume of canned meat.

### CODE REVIEW

**Comment by brannamm** [comment]
Jenkins rebuild

**Comment by brannamm** [comment]
Jenkins rebuild

**Comment by RarkGrames** [comment]
Might be a better solution to half the volume of chunk of meat (which is the basis of every meat product). Currently 296 g meat has 1 L volume, and the density of meat is [around 1](https://www.sciencedirect.com/science/article/pii/S0023643801907625).

**Comment by brannamm** [comment]
> Might be a better solution to half the volume of chunk of meat (which is the basis of every meat product). Currently 296 g meat has 0.5 L volume, and the density of meat is [around 1](https://www.sciencedirect.com/science/article/pii/S0023643801907625).

I agree that seems like the best long term solution but it seems like that will be a big overhaul. My change just makes everything consistent without changing everything. Maybe we should make another issue for changing the volume of meat across the board

**Comment by RarkGrames** [comment]
> a big overhaul.
What do you mean? Every cooked meat product (cooked meat, jerky, canned meat rehydrated meat, etc.) inherits its values from raw meat. If you change the volume of chunk of meat, they all will copy the volume from it.

**Comment by brannamm** [comment]
I was assuming that changing the base meat volume would result in existing recipes not fitting their containers but if what you listed is all there is then I will just go ahead with that tonight or tomorrow

**Comment by RarkGrames** [comment]
After going through the whole list, the only meat dish I've found that gets incorrectly modified is the pickled meat in glass jar. There will be 1/2 portions in a jar instead of 1/1. But that is the same case for pickled offal, which is already like that, so they will have to be addressed separately. I will make a PR fixing them if you do this and it gets merged.

**Comment by brannamm** [comment]
Thanks for the analysis RarkGrames, I reverted my original changes, halved the volume of chunk of meat, and fixed pickled meat, pickled offal, canned human recipes.

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "name_plural": "chunks of meat",
     "description": "Freshly butchered meat.  You could eat it raw, but cooking it is better.",
     "weight": 296,
-    "volume": 2,
+    "volume": 1,
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/carnivore.json]:
```suggestion
    "volume": "250 ml",
```

**Comment by brannamm** [comment]
Updated the meat cooling unit test since meat now has less volume thus less thermal conductivity and the test relies on guesstimated times and temperature measurements

**Pull request merged**